src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -737,7 +737,7 @@ private List<ModificationInfos> execCreateEquipmentDeletion(NetworkStoreListener
                 }
 
                 if (identifiable instanceof Connectable) {
-                    ((Connectable) identifiable).remove(true);
+                    new RemoveFeederBay(equipmentId).apply(network, false, reporter);
```

Comment:
true ?

Reply:
As a general rule, we just want to report failure in reporter, right?